.build/check-code.sh

```diff
@@ -24,5 +24,5 @@ command -v ant >/dev/null 2>&1 || { echo >&2 "ant needs to be installed"; exit 1
 [ -f "${CASSANDRA_DIR}/build.xml" ] || { echo >&2 "${CASSANDRA_DIR}/build.xml must exist"; exit 1; }
 
 # execute
-ant -f "${CASSANDRA_DIR}/build.xml" check dependency-check
+ant -f "${CASSANDRA_DIR}/build.xml" check # dependency-check # FIXME dependency-check now requires NVD key downloaded first
```

Comment:
we could probably workaround it somewhere when this gets merged.

https://github.com/apache/cassandra/pull/3191/files

We might stop to require key if we just point it to a mirror.

Reply:
@michaelsembwever Is it urgent/important? I could investigate.

Reply:
This is related to the following issue:
https://issues.apache.org/jira/browse/CASSANDRA-19146

The NVD database should be cached somewhere on the agent's side (~94,6 MB on disk on my local machine), however, I'm not sure that it is an appropriate thing for Jenkins agents.